libraries/chain/controller.cpp

```diff
@@ -2830,6 +2830,11 @@ block_id_type controller::last_irreversible_block_id() const {
    return get_block_id_for_num( lib_num );
 }
 
+block_timestamp_type controller::last_irreversible_block_timestamp() const {
```

Comment:
Call this `last_irreversible_block_time` and return a `time_point` to be consistent with the other ones.

Reply:
Done. I used `.to_time_point()` since I dislike auto type conversion. Maybe my days with Ada programming language coming out, but really because I've been burned too many times.